frameworks/keyed/angular-cf-signals/src/main.ts

```diff
@@ -1,4 +1,9 @@
 import { bootstrapApplication } from '@angular/platform-browser';
 import { AppComponent } from './app/app.component';
+import { provideExperimentalZonelessChangeDetection } from '@angular/core';
 
-bootstrapApplication(AppComponent);
+bootstrapApplication(AppComponent, {
+    providers: [
+      provideExperimentalZonelessChangeDetection(),
```

Comment:
I would suggest reverting for now based on the discussion in https://github.com/pkozlowski-opensource/js-framework-benchmark/commit/307580b3c4bcd1cd83aef10bef3f075f3cd7f20f

Reply:
Ok. But then it seems like it have to add zone.js to angular.json again.

Reply:
Yes, in this benchmark, yes.